libnmstate/nm/sriov.py

```diff
@@ -72,8 +140,63 @@ def get_info(device):
     )
 
     if sriov_setting:
-        info[Ethernet.SRIOV_SUBTREE] = {
-            Ethernet.SRIOV.TOTAL_VFS: sriov_setting.props.total_vfs
-        }
+        sriov_config = {}
+        sriov_config[Ethernet.SRIOV.TOTAL_VFS] = sriov_setting.props.total_vfs
+
+        vfs_config = _create_sriov_vfs_config(sriov_setting)
+        sriov_config[Ethernet.SRIOV.VFS_SUBTREE] = vfs_config
+
+        info[Ethernet.SRIOV_SUBTREE] = sriov_config
```

Comment:
I think we should provide the info from the kernel (sysfs in this case) and open an RFE on NM to provide this data through libnm.

Reply:
That is okay for `total-vfs` but not for other VF options because it have different formats. I suggest to get `total-vfs` from `/sys/class/net/..` but get the VF options from NM. The RFE is already opened.

Reply:
Done for `total-vfs`. Thanks!

Reply:
> not for other VF options because it have different formats.
> but get the VF options from NM

I do not understand this point.
If you will not fetch this info from the kernel, then you cannot report the state of all available VF/s that exists.

Reply:
We have decided to parse the `ip link show` output in a following PR.